sdk/core/src/CommonNetworkMember/BaseNetworkMember/RegisteringService.ts

```diff
@@ -0,0 +1,79 @@
+import { ParsedOptions } from '../../shared/getOptionsFromEnvironment'
```

Comment:
It belongs to the `sdk/core/src/services` directory.

Reply:
^^
Due to I ope in the future we can refactor this class with 1100 lines of code, so it makes sense to keep all it parts in one place